<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WizardWelcome</class>
 <widget class="QWizard" name="WizardWelcome">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wizard</string>
  </property>
  <widget class="QWizardPage" name="wizardPage1">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="radioButtonNew">
      <property name="text">
       <string>Новый эксперимент</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButtonLoad">
      <property name="text">
       <string>Загрузить конфигурацию эксперимента</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="wizardPage2">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="labelWallCount">
      <property name="text">
       <string>Количество стен</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QLabel" name="labelCoeff">
      <property name="text">
       <string>Коэффициент рассеивания</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="labelFilename">
      <property name="text">
       <string>Имя файла</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLineEdit" name="lineEditFilename">
      <property name="placeholderText">
       <string>По умолчанию default.mirror</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="pushButtonSearch">
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Обзор</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2" colspan="2">
     <widget class="QLineEdit" name="lineEditCoeff">
      <property name="placeholderText">
       <string>Можно будет изменить</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2" colspan="2">
     <widget class="QLineEdit" name="lineEditWallCount">
      <property name="placeholderText">
       <string>От 4 до 9</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
